.travis.yml

```diff
@@ -3,4 +3,4 @@ install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/ma
 script: bash -ex .travis-opam.sh
 sudo: true
 env:
-        - OCAML_VERSION=4.02 PACKAGE=xapi-idl EXTRA_REMOTES=git://github.com/xapi-project/opam-repo-dev
+        - OCAML_VERSION=4.02 PACKAGE=xapi-idl EXTRA_REMOTES=git://github.com/xapi-project/xs-opam
```

Comment:
It is picking up cohttp 0.22.0 from some upstream repository, while xs-opam providex 0.21.0 and uchar. I believe Uchar became part of ocaml stdlib in a more recent release.

Can we make sure that the travis opam does not use any upstream repository other than the one in xs-opam?